spec/moonshot/build_mechanism/github_release_spec.rb

```diff
@@ -189,5 +189,32 @@ module Moonshot # rubocop:disable ModuleLength
         expect { subject.send(:check_ci_status, sha) }.to raise_error(Shell::CommandError)
       end
     end
+    
+    
+    describe '#validate_commit' do
+      subject do
+        s = described_class.new(build_mechanism , skip_ci_status: true)
+        s.resources = resources
+        s
+      end
+    
+      let(:step) do
+        instance_double(InteractiveLogger::Step)
+      end
+
+      before(:each) do
+        allow(resources.ilog).to receive(:start_threaded).and_yield(step)
+        Retriable.configure do |c|
+          c.sleep_disabled = true
+        end
+      end
+
+      it 'passes if the validate commit return nil with skip_ci_status flag' do
```

Comment:
I feel like it would be better to explicitly assert that check_ci_status is not run when the flag is set.  But I don't feel strongly.

Reply:
Yeah, I think that's a good test, and it keeps the unit boundaries. In the current implementation the details of check_ci_status matter, and they shouldn't.

Reply:
Fixed